### PR TITLE
Bestblocksending

### LINKED ISSUE #62: BlockchainDownloader doesnt download blocks if height of blockchain more than 100
BlockchainDownloader doesnt download blocks if height of blockchain more than 100

### PR DESCRIPTION
**Pull request by Parabellum1905y**
fixed #62 , #59 , #56

### CODE REVIEW

**Comment by Parabellum1905y** [comment]
fixed #53 with 8de404bdb44202a8768df511772e31caead6ce3f commit

File: src/main/scala/network/BlockchainDownloader.scala [hunk 1]
```diff
@@ -255,3 +259,21 @@ object BlockchainDownloader {
 
    case class DownloadEnded(hisReader : Option[AeneasHistory]) extends DownloaderEvent
 }
+
+trait DownloaderHistoryAppender[B<:AeneasBlock, H<:History[B, _, _]]{
```
Comment (Fly-Style) [inline src/main/scala/network/BlockchainDownloader.scala]: Code style : don't forget, please, about spaces.

File: src/test/scala/setup/PreMiningSetupTest.scala [hunk 2]
```diff
@@ -17,7 +17,7 @@ import scala.concurrent.duration._
   */
 class PreMiningSetupTest extends FunSuite with Matchers {
    test("Mining time setup test") {
-      implicit val currentViewTimer = 70.second
+      implicit val currentViewTimer = 80.second//TODO надо обсудить эту величину, а было 70
```
Comment (Fly-Style) [inline src/test/scala/setup/PreMiningSetupTest.scala]: This is heuristics value.

**Comment by Fly-Style** [comment]
Closed.

**Pull request merged**